Slaves/Slave.js

```diff
@@ -80,7 +80,8 @@ function addPunishmentPoints(amount) {
     let multiplier = 1;
 
     //TODO: Base on amount of recent pp too?
-    if(isVar(VARIABLE_LAST_PUNISHMENT_POINT_CHANGE)) {
+	//adding a check for a nonexistnat variable to turn off wacky punishment points (should probably be an exposed setting)
```

Comment:
What? Why wacky pp?

Reply:
you call a function with a specific value specified as a parameter... and that value doesnt get assigned.... sometimes it whimsically hands out thousands of punishment points (ouch!)  

maybe you should reimpliment the function to assign the parameter if there is one, and use your algorithm if its called without a parmeter?

Reply:
(ie istr there are legacy places in code that already decide based on variables how many points to assign prior to making the call)